recipes/intel-ipsec-mb/all/conanfile.py

```diff
@@ -57,8 +57,11 @@ def validate(self):
         if self.settings.arch != "x86_64":
             message = "{} is not supported".format(self.settings.arch)
             raise ConanInvalidConfiguration(message)
+        if self.settings.os not in ("Linux", "Windows"):
```

Comment:
https://github.com/intel/intel-ipsec-mb#freebsd-64-bit-only

```suggestion
        if self.settings.os not in ("FreeBSD", "Linux", "Windows"):
```

Reply:
It seems it requires `gmake` in the path. I am not sure if the current recipe can package it.

Reply:
Are you testing on FreeBSD?

Reply:
No, I am not. I only have a Linux machine.

Reply:
conan will use `make` when doing `autotools.make()`.
Almost all recipes assume `make` to be gnumake compatible.

Reply:
We can also set `make_program=gmake`. This was also the case with `nmake` I guess. It seems it can [handle it](https://github.com/conan-io/conan/blob/c1ab505b80c41567b35e8b5cbc0e00fba2e5c11d/conans/client/build/autotools_environment.py#L241) nicely.

Reply:
I currently use Ubuntu and it provides `/usr/bin/make`. I have no problem on my distro. I am just worried about FreeBSD and Windows because I can't test the recipe on these systems.

Reply:
It's okay if a recipe doesn't work on a non-supported platform.
A freebsd user can always fix it later.